### PR TITLE
Math magicians events

### PR DESCRIPTION
**Pull request by eerapheal**
This is my Math App project. this component was dynamically built with the following:

- npx create-react-app
- HTML
- CSS
- JS

### CODE REVIEW

File: src/component/calculator.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  render() {
+    const {total, next, operation } = this.state;
+    return (
+      <div className="container">
+        <div className="mainDiv">
+          <button className="inputiv" type="submit">{total}{operation}{next}</button>
+          <button className="btn grey" onClick={this.handleClick} type="submit">AC</button>
+          <button className="btn grey" onClick={this.handleClick} type="submit">+/-</button>
+          <button className="btn grey" onClick={this.handleClick} type="submit">%</button>
+          <button className="btn orange" onClick={this.handleClick} type="submit">/</button>
```
Comment (danielufeli) [inline src/component/calculator.js]: - [ ] Nice working on your operators, kindly check I get the error in the screenshot below when trying to divide you can use the `÷` symbol instead 
image

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @eerapheal , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of GitHub flow ✔️
- Clean Code✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/component/calculator.js (hunk 1 above)
Comment (eerapheal) [reply src/component/calculator.js]: Okay, Thanks.

**Review by jaflih — APPROVED** [review]
Hi @eerapheal   , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

Good Readme ✔️
No linter error ✔️
Good PR title ✔️
Good PR summary ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @jaflih in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**